docs/_docs/pagination.md

```diff
@@ -207,7 +207,7 @@ page with links to all but the current page.
     {% if page == paginator.page %}
       <em>{{ page }}</em>
     {% elsif page == 1 %}
-      <a href="{{ paginator.previous_page_path | prepend: site.baseurl | replace: '//', '/' }}">{{ page }}</a>
+      <a href="{{ "/index.html" | prepend: site.baseurl | replace: '//', '/' }}">{{ page }}</a>
```

Comment:
Each user will need to customize his/her pagination I think. So that makes my example no better than yours.